engine/src/bridge_converter.rs

```diff
@@ -292,6 +280,20 @@ impl<'a> BridgeConverter {
         }
     }
 
+    fn get_blank_extern_c_mod(&self) -> ItemForeignMod {
+        ItemForeignMod {
+            attrs: Vec::new(),
+            abi: syn::Abi {
+                extern_token: Token![extern](Span::call_site()),
+                name: Some(syn::LitStr::new("C", Span::call_site())),
+            },
+            brace_token: syn::token::Brace {
+                span: Span::call_site(),
+            },
+            items: Vec::new(),
+        }
```

Comment:
These syntax tree nodes are not really designed to be instantiated other than by a parser. [syn::parse_quote](https://docs.rs/syn/1.0.42/syn/macro.parse_quote.html) would be a good fit here:
<br>

```suggestion
        parse_quote!(extern "C" {})
```

Reply:
Thanks - over time I'll gradually address this and replace them all with `parse_quote`. Filed #12 to remind me.